lightning/src/onion_message/functional_tests.rs

```diff
@@ -170,3 +170,13 @@ fn reply_path() {
 		"lightning::onion_message::messenger".to_string(),
 		format!("Received an onion message with path_id: None and reply_path").to_string(), 2);
 }
+
+#[test]
+fn peer_buffer_full() {
+	let nodes = create_nodes(2);
+	for _ in 0..188 {
```

Comment:
Consider making the magic number a function of `MAX_PER_PEER_BUFFER_SIZE`?

Reply:
I kinda like keeping that `const` private within the method

Reply:
Could you add a comment in lieu of that?

Reply:
Yes, will do